content/cli/usages.md

```diff
@@ -1,169 +1,169 @@
-### CLI command reference
+### Référence des commandes de l'interface de ligne de commande (CLI)
 
 #### nest new
 
-Creates a new (standard mode) Nest project.
+Créez un nouveau projet Nest (mode standard).
 
 ```bash
-$ nest new <name> [options]
-$ nest n <name> [options]
+$ nest new <nom> [options]
+$ nest n <nom> [options]
 ```
 
 ##### Description
 
-Creates and initializes a new Nest project. Prompts for package manager.
+Créez et initialisez un nouveau projet Nest. Prompts pour le gestionnaire de paquets.
```

Comment:
Prompts for... = Demande le... (en effet ici il faut choisir entre yarn, npm, etc)

Reply:
Demande de choisir le gestionnaire de paquets?

Reply:
J'ai mis "demande de choisir" si tu préfère sans "choisir" ou ajouter qqch dis moi

Reply:
ok pour moi !